packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -0,0 +1,323 @@
+
+8. Okta sends back the tokens.
+
+## Configure CIBA for your org
+
+There are a few steps required to configure CIBA for your org:
+
+> **Note:** Ensure that you complete the **What you need** section at the beginning of this guide before proceeding.
+
+* Create an OIDC client integration and enable the CIBA grant type.
+* Configure an authorization server access policy to use CIBA.
+
+### Create an OIDC client integration and enable CIBA
+
+Create an Okta OIDC client app integration to represent the consumption device, enable CIBA as a grant type, and associate the client app with the [custom authenticator that you previously configured using the Devices SDK](https://github.com/okta/okta-mobile-swift).
+
+> **Note:** Web apps are currently the only application supported with the CIBA flow.
+
+1. In the Admin Console, go to **Applications** > **Applications**.
+2. Click **Create App Integration**.
+3. Select **OIDC - OpenID Connect** as the **Sign-in method** and choose which type of application that you want to integrate with Okta. In this use case example, we use **Web Application**.
+4. Click **Next**.
+5. Enter a name for your app integration.
+6. Select the **Client-initiated backchannel authentication flow (CIBA)** checkbox as a **Grant type**.
+7. Select the custom authenticator that you [previously configured using the Mobile SDK](https://github.com/okta/okta-mobile-swift) from the **Preferred authenticator for CIBA** dropdown list.
+8. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns the tokens after the user finishes authenticating. You can use the default URI for this exercise.
+
+    > **Note:** You can leave the rest of the default values, as they work with this guide for testing purposes.
+
+9. In the **Assignments** section, select **Allow everyone in your organization to access**.
+10. Click **Save**. The settings page for the app integration appears, showing the **General** tab.
+11. Make note of the **Client ID** in the **Client Credentials** section and the client secret in the **CLIENT SECRETS** section. You need this information for the [Test the CIBA flow](#test-the-ciba-flow) section.
+
+#### Use the API
+
+You can also use the [Apps API](/docs/reference/api/apps/#add-oauth-2-0-client-application) to create the OAuth 2.0 client app with the CIBA grant type. Use the following parameters in the request:
+
+**Supported parameters**
+
+| Parameter                    | Value                                                                        |
+| :--------------------------- | :--------------------------------------------------------------------------- |
+| `grant_types`                | `urn:openid:params:grant-type:ciba`     |
+| `backchannel_authentication_request_signing_alg` | (Optional) The signing algorithm for CIBA signed requests using JWT. If this value isn't set and a JWT signed request is sent, the request fails. |
```

Comment:
If the request fails, why is this value optional? @mymoonasaifunnissameerasahib-okta